gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,63 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\]'
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Sender -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\}(?:,\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\})*\]'
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Receiver -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\}(?:,\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\})*\]'
+
+-- ee.gno --
+package ee
+
+import (
+	"std"
+)
+
+const (
+	EventFoo = "foo"
+	EventBar = "bar"
+)
+
+func Sender(){
+    SubSender()
```

Comment:
foo, bar...

Reply:
Agreed. @notJoon If you disagree on some items, don't resolve the conversation but leave it open so the original author can agree/disagree with you.

We can probably add this to CONTRIBUTING.

Reply:
We actually have this already in the CONTRIBUTING:
https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md#how-do-i-submit-changes